assemblies/DataSources/OpenConfig/Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.xml

```diff
@@ -0,0 +1,996 @@
+             </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient.MyServicesGroupType">
+            <summary>
+            The services group type for the instance of this base class.
+            Helps to build the correct MessageBroker subjects.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient.ChannelIdentifier">
+            <summary>
+            The channel ID helps identify the source when investigating logging and traces.
+            It also makes the subjects used to communicate over the message broker unique.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient.ChannelId">
+            <inheritdoc />
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient.ChannelName">
+            <inheritdoc />
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient.InternalLogger">
+            <summary>
+            The logger instance that will not throw exceptions.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient.Logger">
+            <inheritdoc />
+        </member>
+        <member name="F:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient._messageProcessor">
+            <summary>
+            Responsible for queueing and processing the incoming NATS messages.
+            </summary>
+        </member>
+        <member name="F:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient._responseHandlers">
+            <summary>
+            Mapping from the command type to the function handling said command.
+            </summary>
+            <remarks>
+            Any new request type for this handler should be added in PrepareResponseHandlers().
+            </remarks>
+        </member>
+        <member name="F:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient._natsDxmTimeoutTimer">
+            <summary>
+            A watchdog timer to check if the DxM on NATS is still sending messages (responses, events, or heartbeats).
+            Will disconnect when breached.
+            </summary>
+        </member>
+        <member name="F:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.AMiddlewareClient._natsHeartbeatRequiredTimer">
+            <summary>
+            A watchdog timer to check if we have been silent for too long and need to send a heartbeat message to the DxM.
```

Comment:
In technical documentation, using "we" is generally avoided. It's more something that is used in academic writing, but it's not very appropriate here. However, to rephrase this, I'd need to know: what is the "we" that is referred to here?